src/app/zap-templates/partials/idl/structure_member.zapt

```diff
@@ -1,7 +1,11 @@
 {{~#if isOptional~}} optional {{/if~}}
 {{~#if isNullable~}} nullable {{/if~}}
 
-{{type}} {{asLowerCamelCase label~}}
+{{type}} {{~#if (isString type)~}}
+  {{~#if maxLength~}} 
+    <{{maxLength}}> 
+  {{~/if~}}
```

Comment:
Just looking at the output, we may be running into the problem where for an array the `type` is the _entry type_...  Gotta love this type system.  So this should probably be:
```suggestion
{{type}} {{~#unless isArray}}{{#if (isString type)~}}
  {{~#if maxLength~}} 
    <{{maxLength}}> 
  {{~/if}}{{/unless~}}
```

Reply:
I did add the check - however with or without it, I saw no output difference.

I am not convinced zap supports "array of strings".